src/main/java/org/candlepin/dto/api/v1/ConsumerArrayElementTranslator.java

```diff
@@ -138,7 +138,7 @@ public ConsumerDTOArrayElement populate(ModelTranslator translator,
             }
 
             Environment environment = this.environmentCurator.getConsumerEnvironment(source);
-            dest.setEnvironment(translator.translate(environment, EnvironmentDTO.class));
+            dest.addEnvironments(translator.translate(environment, EnvironmentDTO.class));
```

Comment:
Same here: we should be using a new environmentCurator method that returns all the environments of a consumer